app/globals.py

```diff
@@ -18,6 +19,15 @@ def get_questionnaire_store(user_id, user_ik):
     return store
 
 
+def get_session_store():
+    # Sets up a single SessionStorage instance throughout app.
```

Comment:
its not throughout the app its per request.

Reply:
Lazy cut and paste from questionnaire_store above. Corrected both.